### ISSUE TITLE
verify text cues work within unity app

### ISSUE THREAD

**Issue by jakemoves** [jakemoves/cohort-unity-client#3]
- CHSession should emit a single event when a text cue is fired with the cue's text
- the cue text should be displayed in a UI element

**Comment by jakemoves** [comment]
OK, this is implemented and will be released on TestFlight for testing in version 0.3.0.

There's a single text cue in the Unity app that can be triggered with this JSON sent to the broadcast endpoint:
```
{ 
	"mediaDomain": 2,
	"cueNumber": 1,
	"cueAction": 0,
	"targetTags": ["all"]
}
```

This requires the text that you want to display on the device to be included in the app build. This is reasonable for media files (sound / video cues) since we don't support streaming content — everything has to be included. But it's a little bit overkill for text. So I also implemented a feature for arbitrary text cues, where you can decide what text you're displaying when you send the cue. Sending this JSON to the /broadcast endpoint should display the `cueContent` text. 

```
{ 
	"mediaDomain": 2,
	"cueNumber": 1,
	"cueAction": 0,
	"targetTags": ["all"],
	"cueContent": "no latin plz"
}
```

Note: even if you specify a valid cue number on the device (i.e. 1, for the demo app) in the JSON, if you include a `cueContent` field it will be used instead of the text defined on the device.

**Comment by AmandaLBaker** [comment]
All text cues available to test via the admin-v2 site (under demo events) send both text and a vibration to the phone. (Vibrate is enabled on device to be able to receive haptic cues, vibration occurs when ringer is on and off)
